### PR TITLE
New package: yuyuanweb.jianqiebanzhushou version 0.0.6

### PR DESCRIPTION
**Pull request by CoolPlayLin**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/151152)

### CODE REVIEW

**Comment by CoolPlayLin** [comment]
https://github.com/microsoft/winget-pkgs/labels/Validation-Domain should be dropped

File: manifests/y/yuyuanweb/jianqiebanzhushou/0.6/yuyuanweb.jianqiebanzhushou.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: yuyuanweb.jianqiebanzhushou
+PackageVersion: "0.6"
```
Comment (stephengillie) [inline manifests/y/yuyuanweb/jianqiebanzhushou/0.6/yuyuanweb.jianqiebanzhushou.installer.yaml:5]:
```suggestion
PackageVersion: 0.0.6
```
Please update file path to match this change.

File: manifests/y/yuyuanweb/jianqiebanzhushou/0.6/yuyuanweb.jianqiebanzhushou.locale.zh-CN.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: yuyuanweb.jianqiebanzhushou
+PackageVersion: "0.6"
```
Comment (stephengillie) [inline manifests/y/yuyuanweb/jianqiebanzhushou/0.6/yuyuanweb.jianqiebanzhushou.locale.zh-CN.yaml:5]:
```suggestion
PackageVersion: 0.0.6
```
Please update file path to match this change.

File: manifests/y/yuyuanweb/jianqiebanzhushou/0.6/yuyuanweb.jianqiebanzhushou.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: yuyuanweb.jianqiebanzhushou
+PackageVersion: "0.6"
```
Comment (stephengillie) [inline manifests/y/yuyuanweb/jianqiebanzhushou/0.6/yuyuanweb.jianqiebanzhushou.yaml:5]:
```suggestion
PackageVersion: 0.0.6
```
Please update file path to match this change.

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @CoolPlayLin,

Should the `PackageVersion` match the Registry, or should it be added as a `DisplayVersion`?

**Comment by CoolPlayLin** [comment]
https://github.com/microsoft/winget-pkgs/labels/Validation-Domain should be dropped

**Comment by stephengillie** [comment]
Verify: 
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**